radix-engine-common/src/math/decimal.rs

```diff
@@ -18,15 +19,20 @@ use crate::math::PreciseDecimal;
 use crate::well_known_scrypto_custom_type;
 use crate::*;
 
-/// `Decimal` represents a 256 bit representation of a fixed-scale decimal number.
+/// `Decimal` represents a 192 bit representation of a fixed-scale decimal number.
 ///
 /// The finite set of values are of the form `m / 10^18`, where `m` is
-/// an integer such that `-2^(256 - 1) <= m < 2^(256 - 1)`.
+/// an integer such that `-2^(192 - 1) <= m < 2^(192 - 1)`.
+///
+/// Fractional part: 60 bits/18 digits
```

Comment:
Can we add `~` because it's not exact 60 bits?

Reply:
Done